tests/test_stack.py

```diff
@@ -705,6 +705,26 @@ def test_validate_template_sends_correct_request(
             kwargs={"Template": sentinel.template}
         )
 
+    @patch("sceptre.stack.Stack.config")
+    @patch("sceptre.stack.Stack._format_parameters")
+    @patch("sceptre.stack.Stack._get_template_details")
+    def test_estimate_template_cost_sends_correct_request(
+        self, mock_get_template_details, mock_format_params, mock_config
+    ):
+        mock_format_params.return_value = sentinel.parameters
+        mock_get_template_details.return_value = {
+            "Template": sentinel.template
+        }
+        self.stack.estimate_template_cost()
+        self.stack.connection_manager.call.assert_called_with(
+            service="cloudformation",
+            command="estimate_template_cost",
+            kwargs={
+                "Parameters": sentinel.parameters,
+                "Template": sentinel.template,
+            }
+        )
+
     @patch("sceptre.stack.Stack._format_parameters")
```

Comment:
Could we have a test when a web browser isn't available for the CLI?

Reply:
added tests to cli where browser call is made

Reply:
don't need to worry